web/src/components/molecules/Content/Form/index.tsx

```diff
@@ -123,7 +123,12 @@ const ContentForm: React.FC<Props> = ({
                 },
               ]}
               name={field.id}
-              label={field.title}>
+              label={
+                <>
+                  <FieldTitle>{field.title}</FieldTitle>
+                  {field.unique ? <FieldUnique>({t("unique")})</FieldUnique> : ""}
+                </>
```

Comment:
To tidy up this component a bit, can you create a small component (can be inside the same file) for `FieldTitle` that takes in a title and isUnique prop and returns this fragment? It is a lot of duplicated code that can be reduced to 
`<FieldTitle title={field.title} isUnique={field.unique} />`